variable_set.go

```diff
@@ -330,6 +379,47 @@ func (s *variableSets) RemoveFromWorkspaces(ctx context.Context, variableSetID s
 	return req.Do(ctx, nil)
 }
 
+// ************************
+// Apply variable set to projects in the supplied list.
+// Note: this method will return an error if the variable set has global = true.
```

Comment:
Since these are not generally available, go ahead and add a note (to be cleaned up later when the feature is GA)

`// **Note: This feature is still in BETA and subject to change.**`